app/views/shared/export/_plan.erb

```diff
@@ -32,55 +32,59 @@
     <% end %>
 
     <% @hash[:phases].each do |phase| %>
-      <div style="page-break-before:always;"></div> <!-- Page break before each phase -->
-      <!-- If there is more than one phase show the plan title and phase title -->
-      <h1><%= (@hash[:phases].length > 1 ? "#{@plan.title} - #{phase[:title]}" : @plan.title) %></h1>
-      <hr />
-      <% phase[:sections].each do |section| %>
-        <% if @show_sections_questions %>
-          <h3><%= section[:title] %></h3>
-        <% end %>
+      <%# Only render selected phase %>
+      <% if phase[:title] == @selected_phase.title %>
+        <div style="page-break-before:always;"></div> <!-- Page break before each phase -->
+        <h1><%= download_plan_page_title(@plan, phase, @hash) %></h1>
+        <hr />
+        <% phase[:sections].each do |section| %>
+          <%# Only render if @show_sections_questions is true and either section not customised or section is customised and @show_custom_sections is true %>
+          <% if @show_sections_questions && (!section[:modifiable] || (@show_custom_sections && section[:modifiable]))  %>
```

Comment:
It would be good to extract this out to a method, in the same way you have `download_plan_page_title`